kcoin/genesis/genesis_generator.go

```diff
@@ -65,7 +65,7 @@ type GenesisOptions struct {
 
 type PrefundedAccount struct {
 	WalletAddress string
-	Balance       int64
+	Balance       string
```

Comment:
why moving to string?

Reply:
I was having problems to manage big numbers from the console or the config files. All the values in the options are of type string, in this case we can benefit too different types of numbers for the configuration as decimal, hex, etc.

After it is converted to big int, address, etc

Reply:
:+1: